integration-tests/src/test/java/com/datastax/oss/driver/core/connection/NettyResourceLeakDetectionIT.java

```diff
@@ -128,7 +128,8 @@ public void should_not_leak_compressed_lz4() {
 
   @BackendRequirement(
       type = BackendType.DSE,
-      description = "Snappy is not supported in OSS C* 4.0+ with protocol v5")
+      minInclusive = "6.0.0",
+      description = "Snappy is not supported in DSE before 6.0 with protocol v3")
```

Comment:
This wording is kinda unclear.  Is this supposed to be v3 or v5 here?  Either way are we saying that there's an explicit dependency on the the DSE version _and_ the protocol version here?  Let's say we are talking about v3... if I use v2 with DSE 6.0 am I okay?

Reply:
I do not know why this was failing for me. Retested now with old DSE version and it worked. Reverted.